sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/api/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.netstandard2.0.cs

```diff
@@ -22,7 +22,8 @@ public partial class AuthenticationEventsTriggerAttribute : System.Attribute
     {
         public AuthenticationEventsTriggerAttribute() { }
         public string AudienceAppId { get { throw null; } set { } }
-        public string TenantId { get { throw null; } set { } }
+        public string AuthorityUrl { get { throw null; } set { } }
```

Comment:
This is a breaking change, and we are okay with it. This will align us with other MSFT auth libraries.